content/blog/using-jina-embeddings-haystack/index.md

```diff
@@ -0,0 +1,198 @@
+
+With the [Jina Haystack extension](https://haystack.deepset.ai/integrations/jina), you can now take advantage of these new text embedders in your Haystack pipelines! In this post, we'll show what's cool about Jina Embeddings v2 and how to use them.
+
+> You can follow along in the accompanying [Colab notebook of a RAG pipeline that uses the Jina Haystack extension](https://colab.research.google.com/drive/1l8GbQhqxnWXkdktgJfs9Rz4EAtNbHK_L#scrollTo=_coq_qCuItbN).
+
+## Advantages of Jina Embeddings v2
+
+- **Handling long documents.** The large token window, accommodating up to 8192 tokens, allows you to break the embeddings into larger chunks. It's more computationally and memory-efficient to use a few larger vectors than a lot of small ones, so this allows Jina v2 to process large documents efficiently. 
+- **Improved semantic understanding.** Larger text chunks also contain more *context* within each chunk, which can help LLMs better understand your documents. Improved understanding means better long document retrieval, semantic textual similarity, text reranking, recommendation, RAG and LLM-based generative search.
+- **Fully open source 💙** There are both small and large embedding models available, depending on your computing resources and requirements. To run the embedding models yourself, [check out this documentation on HuggingFace](https://huggingface.co/jinaai/jina-embeddings-v2-base-en).  Alternately, you can use Jina's fully managed embedding service to handle that for you, which we'll be doing for this demo.
+
+## Getting started using Jina Embeddings v2 with Haystack
+
+To use the integration you'll need a free Jina api key - get one [here](https://jina.ai/embeddings/). 
+
+You can use Jina Embedding models with two Haystack components: `JinaTextEmbedder` and `JinaDocumentEmbedder`.
+
+To create semantic embeddings for documents, use`JinaDocumentEmbedder` in your indexing pipeline. For generating embeddings for queries, use `JinaTextEmbedder`. 
+
+In the following code we'll demonstrate how to use both components. You can also [see the Haystack docs for some minimum viable code examples.](https://haystack.deepset.ai/integrations/jina)
+
+
+## Summarizing legal text with a Haystack RAG pipeline
+
+I'm not a lawyer, and neither are large language models. But LLMs are good at analyzing long, complex documents. So let's try using the Jina v2 embedding models for some legal summarization.
+
+In October 2023, I narrowly escaped jury duty. I had slight FOMO since the case sounded interesting (Google v. Sonos). Let's see how it turned out.
+
+To follow along with this demo, in addition to a Jina api key you'll also need a [Hugging Face access token](https://huggingface.co/docs/hub/security-tokens), since we'll use the [Mixtral 8x7b LLM](https://mistral.ai/news/mixtral-of-experts/) for question answering.
+
+
+First, let's install all the packages we'll need.
+
+```bash
+pip haystack-ai jina-haystack chroma-haystack pypdf
```

Comment:
I think you don't need to install `haystack-ai` additionally, as it comes with both integrations as a dependency
```suggestion
pip install jina-haystack chroma-haystack pypdf
```